packages/integration-karma/test/light-dom/slotting/index.spec.js

```diff
@@ -1,55 +1,104 @@
+        expect(Array.from(nodes['x-light-container'].childNodes)).toEqual([
+            nodes['container-upper-slot-top'],
+            nodes['container-upper-slot-default'],
+            nodes['container-upper-slot-bottom'],
+            nodes['container-default-slot-top'],
+            nodes['1'],
+            nodes['2'],
+            nodes['3'],
+            nodes['4'],
+            nodes['5'],
+            nodes['container-default-slot-bottom'],
+            nodes['container-lower-slot-top'],
+            nodes['container-lower-slot-default'],
+            nodes['container-lower-slot-bottom'],
+        ]);
+
+        nodes.button.click();
+        await Promise.resolve();
+
+        expect(Array.from(nodes['x-light-container'].childNodes)).toEqual([
+            nodes['container-upper-slot-top'],
+            nodes['container-upper-slot-default'],
+            nodes['container-upper-slot-bottom'],
+            nodes['container-default-slot-top'],
+            nodes['5'],
+            nodes['4'],
+            nodes['3'],
+            nodes['2'],
+            nodes['1'],
+            nodes['container-default-slot-bottom'],
+            nodes['container-lower-slot-top'],
+            nodes['container-lower-slot-default'],
+            nodes['container-lower-slot-bottom'],
+        ]);
     });
-});
 
-describe('imperative slotted content', () => {
-    beforeEach(() => {
-        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', true);
+    it('shadow container, light consumer', () => {
+        const nodes = createTestElement('x-light-consumer', LightConsumer);
+
+        const expected = process.env.NATIVE_SHADOW // native shadow doesn't output slots in innerHTML
+            ? '<x-shadow-container><p data-id="light-consumer-text">Hello from Light DOM</p></x-shadow-container>'
+            : '<x-shadow-container><slot><p data-id="light-consumer-text">Hello from Light DOM</p></slot></x-shadow-container>';
+        expect(nodes['x-light-consumer'].innerHTML).toEqual(expected);
     });
 
-    afterEach(() => {
-        setFeatureFlagForTest('ENABLE_LIGHT_DOM_COMPONENTS', false);
+    it('light container, shadow consumer', () => {
+        const nodes = createTestElement('x-shadow-consumer', ShadowConsumer);
+
+        expect(nodes['x-shadow-consumer'].shadowRoot.innerHTML).toEqual(
+            '<x-light-container><p data-id="container-upper-slot-top">Upper slot top</p><p data-id="container-upper-slot-default">Upper slot default</p><p data-id="container-upper-slot-bottom">Default slot bottom</p><p data-id="container-default-slot-top">Default slot top</p><p data-id="shadow-consumer-text">Hello from Shadow DOM</p><p data-id="container-default-slot-bottom">Default slot bottom</p><p data-id="container-lower-slot-top">Lower slot top</p><p data-id="container-lower-slot-default">Lower slot default</p><p data-id="container-lower-slot-bottom">Lower slot bottom</p></x-light-container>'
```

Comment:
What is the original intention of this test? Could it be simplified to avoid having to maintain this large HTML string?